pkg/controller/releasemanager/incarnation.go

```diff
@@ -171,7 +171,11 @@ func (i *Incarnation) retire() error {
 }
 
 func (i *Incarnation) schedulePermitsRelease() bool {
-	return schedulePermitsRelease(time.Now(), i.target().Release.Schedule)
+	if i.revision == nil {
+		return false
+	}
+	t := i.revision.GetCreationTimestamp().Time
+	return schedulePermitsRelease(t, i.target().Release.Schedule)
```

Comment:
I think this is good for the end of the day and bad for the start of the day. this won't restart releases until there's a new `Revision`, right?

Reply:
lol, good catch!